contributors/guide/github-workflow.md

```diff
@@ -7,38 +7,41 @@ description: |
   sync with upstream and commit hygiene.
 ---
 
+This document is an overview of the GitHub workflow used by the
+Kubernetes project. It includes tips and suggestions on keeping your
+local environment in sync with upstream and how to maintain good
+commit hygiene.
```

Comment:
similar to the above, if it can go in the description field, that would be stellar 🙏